<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserForm</class>
 <widget class="QDialog" name="UserForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>328</width>
    <height>353</height>
   </rect>
  </property>
  <property name="Новый пользователь" stdset="0">
   <string>MainWindow</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Логин</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLineEdit" name="line_login"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Пароль</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QLineEdit" name="line_password"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Фамилия</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QLineEdit" name="line_surname"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Имя</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="3">
    <widget class="QLineEdit" name="line_name"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Отчество</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="3">
    <widget class="QLineEdit" name="line_middleName"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Звание</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Кафедра</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="3">
    <widget class="QComboBox" name="cb_department">
     <item>
      <property name="text">
       <string>-</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Должность</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="3">
    <widget class="QComboBox" name="cb_post">
     <item>
      <property name="text">
       <string>-</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Кафедра 2</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1" colspan="3">
    <widget class="QComboBox" name="cb_department_2">
     <item>
      <property name="text">
       <string>-</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Должность 2</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1" colspan="3">
    <widget class="QComboBox" name="cb_post_2">
     <item>
      <property name="text">
       <string>-</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="10" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>72</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="10" column="1">
    <widget class="QPushButton" name="btn_register">
     <property name="text">
      <string>Регистрация</string>
     </property>
    </widget>
   </item>
   <item row="10" column="2">
    <widget class="QPushButton" name="btn_cancel">
     <property name="text">
      <string>Отмена</string>
     </property>
    </widget>
   </item>
   <item row="10" column="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>51</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="1" colspan="3">
    <widget class="QComboBox" name="cb_rang"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
